### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by sviridovich**

### CODE REVIEW

File: src/main/java/com/onseo/courses/ds/interfaces/QuizController.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.onseo.courses.ds.interfaces;
+
+import com.onseo.courses.ds.controllers.QuizOpenResponse;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@RequestMapping("/api")
+public interface QuizController {
```
Comment (SergiiKravchuk) [inline src/main/java/com/onseo/courses/ds/interfaces/QuizController.java]: interface for QuizController already exists. Use existing one or update it.

Comment (SergiiKravchuk) [inline src/main/java/com/onseo/courses/ds/interfaces/QuizController.java]: interface for QuizController already exists. Use existing one or update it.

File: src/test/java/com/onseo/courses/ds/controllers/QuizControllerImplTest.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+@WebMvcTest(QuizControllerImpl.class)
+public class QuizControllerImplTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Test
+    public void getOpenQuiz() throws Exception {
+        mvc.perform( MockMvcRequestBuilders
+                .get("/api/quiz/open/{quizAssignmentID}", "assignment_ID")
```
Comment (SergiiKravchuk) [inline src/test/java/com/onseo/courses/ds/controllers/QuizControllerImplTest.java]: Not sure about this line/test.

Comment (SergiiKravchuk) [inline src/test/java/com/onseo/courses/ds/controllers/QuizControllerImplTest.java]: Not sure about this line/test.

File: src/test/java/com/onseo/courses/ds/controllers/QuizControllerImplTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(QuizControllerImpl.class)
+public class QuizControllerImplTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Test
+    public void getOpenQuiz() throws Exception {
```
Comment (MaksymChervinskyi) [inline src/test/java/com/onseo/courses/ds/controllers/QuizControllerImplTest.java]: rename to getOpenQuizTest

**Pull request merged**